### DISCUSSION TITLE
Appium 8, custom WebElement - how to?

### DISCUSSION THREAD

**Discussion by DzmitryIlyin** [appium/java-client#1642] [Q&A]
Hi,
I'm wondering if there's some example on how to add custom WebElement using latest Appium on Java?
I assume that it should be done by extending AppiumFieldDecorator and passing customised DefaultFieldDecorator there, but not sure where to start from.
Any help will be much appreciated :)

**Comment by mykola-mokhnach** [comment]
Try asking at https://discuss.appium.io
